components/grid/grid.html

```diff
@@ -0,0 +1,76 @@
+<!DOCTYPE html>
+<html lang="en">
+
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <link rel="stylesheet" href="../../main.css" />
+    <title>Modal</title>
+</head>
+
+<body>
+    <div class="modal-wrapper p-l-xxl p-r-xxl p-b-xxl">
+        <h1>GRID</h1>
+        <p class="text-l fw-l">
+            Grid is used to easily create responsive layouts.
+        </p>
+        <h2>Get Started</h2>
+        <p>
+            To use only the modal component in your project. Kindly include the following CDN in your HTML file.
+        </p>
+        <div class="iframe-container flex-r flex-center m-tb-l">
+            <iframe class="iframe-codes cdn-link"
+                src="https://carbon.now.sh/embed?bg=rgba%28171%2C+184%2C+195%2C+1%29&t=seti&wt=none&l=auto&width=680&ds=true&dsyoff=20px&dsblur=68px&wc=true&wa=true&pv=56px&ph=56px&ln=false&fl=1&fm=Hack&fs=14px&lh=133%25&si=false&es=2x&wm=false&code=%250A%253Clink%2520rel%253D%2522stylesheet%2522%2520href%253D%2522https%253A%252F%252Fui-phoenix.netlify.app%252Fcomponents%252Fgrid%252Fgrid.css%2522%253E%250A"
+                sandbox="allow-scripts allow-same-origin">
+            </iframe>
+        </div>
+        <div class="modal-parent-container">
+            <h2>Grid-2</h2>
+            <p class="text-sm">
+                For having two items in a grid simply copy the code below and paste it in your project.
+            </p>
+
+            <div class="grid-container p-xxl flex-c flex-center position-r br-md m-t-xl">
+                <div class="grid-two p-md">
+                    <span class="item">Item 1</span>
+                    <span class="item">Item 2</span>
+                </div>
+            </div>
+            <h3>Code Snippit</h3>
+            <div class="iframe-container flex-r flex-center m-tb-l">
+
+                <iframe class="iframe-codes"
+                    src="https://carbon.now.sh/embed?bg=rgba%28171%2C+184%2C+195%2C+1%29&t=seti&wt=none&l=auto&width=680&ds=true&dsyoff=20px&dsblur=68px&wc=true&wa=true&pv=56px&ph=56px&ln=false&fl=1&fm=Hack&fs=14px&lh=133%25&si=false&es=2x&wm=false&code=%253Cdiv%2520class%253D%2522grid-two%2520p-md%2522%253E%250A%2520%2520%2520%2520%2520%2520%253Cspan%2520class%253D%2522item%2522%253EItem%25201%253C%252Fspan%253E%250A%2520%2520%2520%2520%2520%2520%253Cspan%2520class%253D%2522item%2522%253EItem%25202%253C%252Fspan%253E%250A%253C%252Fdiv%253E"
+                    style="width: 471px; height: 260px; border:0; transform: scale(1); overflow:hidden;"
+                    sandbox="allow-scripts allow-same-origin">
+                </iframe>
+            </div>
+            <h2>Grid-3</h2>
+            <p class="text-sm">
+                For having two items in a grid simply copy the code below and paste it in your project.
```

Comment:
I think you meant three items here, otherwise  rest of it is okay.

Reply:
will change that !